src/Analyzers/Core/CodeFixes/GenerateConstructor/AbstractGenerateConstructorService.State.cs

```diff
@@ -19,12 +19,6 @@
 using Microsoft.CodeAnalysis.Shared.Utilities;
 using Roslyn.Utilities;
 
-#if CODE_STYLE
-using DeclarationModifiers = Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers;
-#else
-using DeclarationModifiers = Microsoft.CodeAnalysis.Editing.DeclarationModifiers;
-#endif
```

Comment:
this was teh core thing i wanted to get rid of.  we have DeclarationModifiers as a public API in teh workspace.  We then also cloned it into an internal namespace, both to get access to internal parts of it *and* to pull it down to our 'analyzer/compiler' layer.    That's awwful.  

I've separated this out so there's a trivial enum now at hte analyzer/compiler layer.  ANd only workspace/fixer/refactoring layer types access the true DeclarationModifiers type.